.gitignore

```diff
@@ -1,3 +1,4 @@
 /.project
 /.settings/
 /target/
+/gradle-scijava/.gradle
```

Comment:
@elect86 I would add `/.idea/` here and remove all the `.idea` files this PR adds right now.